squid-blockexplorer/src/mocks/mocks.ts

```diff
@@ -0,0 +1,191 @@
+import { Chain } from "@subsquid/substrate-processor/lib/chain";
+import { Logger } from "@subsquid/logger";
+import { Store } from "@subsquid/typeorm-store";
+import { CallItem, Context, EventItem } from '../processor';
+import BlockHeaderMock from './BlockHeader.json';
+import { SubspaceRecordsRootStorage, SubspaceSolutionRangesStorage } from '../types/storage';
+import { Account } from '../model';
+import { createBlock, createCall, createExtrinsic } from '../blocks/utils';
+
+const callData = {
+  id: 'parent call id',
+  name: 'Pallet.parent_call',
+  args: {},
+  pos: 1,
+  parent: undefined,
+  origin: {
+    value: {
+      value: "0x14682f9dea76a4dd47172a118eb29b9cf9976df7ade12f95709a7cd2e3d81d6c",
+      __kind: 'Signed',
+    },
+    __kind: 'system',
+  },
+  success: true,
+  error: undefined,
+}
+
+const extrinsicData = {
+  id: 'parent call extrinsic id',
+  indexInBlock: 0,
+  version: 3,
+  success: true,
+  hash: 'random extrinsic hash',
+  pos: 1,
+  call: callData,
+}
+
+export const callItemWithSignature = {
+  call: callData,
+  extrinsic: {
+    ...extrinsicData,
+    signature: {
+      address: 'random address',
+      signature: {
+        value: 'random signature',
+      },
+      signedExtensions: {},
+    }
+  },
+  name: 'Pallet.parent_call',
+  kind: 'call',
+} as CallItem;
+
+export const parentCallItem = {
+  call: callData,
+  extrinsic: extrinsicData,
+  name: 'Pallet.parent_call',
+  kind: 'call',
+} as CallItem;
+
+export const childCallItem = {
+  name: 'Pallet.child_call',
+  kind: 'call',
+  extrinsic: extrinsicData,
+  call: {
+    ...callData,
+    name: 'Pallet.child_call',
+    id: 'child call id',
+    parent: callData
+  }
+} as unknown as CallItem;
+
+export const callsItems = [
+  {
+    ...callItemWithSignature,
+    call: { ...callItemWithSignature.call, id: 'first id' },
+    extrinsic: { ...callItemWithSignature.extrinsic, id: 'first id' }
+  } as CallItem,
+  {
+    ...callItemWithSignature,
+    call: { ...callItemWithSignature.call, id: 'second id' },
+    extrinsic: { ...callItemWithSignature.extrinsic, id: 'second id' }
+  } as CallItem,
+];
+
+export const eventItemWithoutExtrinsic = {
+  name: 'Random Event name',
+  kind: 'event',
+  event: {
+    id: 'random event id',
+    extrinsic: undefined,
+    call: parentCallItem.call,
+    phase: 'ApplyExtrinsic',
+    name: 'Random Event name',
+    indexInBlock: 0,
+    args: {},
+    pos: 1,
+  }
+} as unknown as EventItem;
+
+export const eventItemWithExtrinsic = {
+  ...eventItemWithoutExtrinsic,
+  event: {
+    ...eventItemWithoutExtrinsic.event,
+    extrinsic: {
+      id: 'parent call extrinsic id',
+      indexInBlock: 0,
+      version: 3,
+      call: parentCallItem.call,
+      success: true,
+      hash: 'random extrinsic hash',
+      pos: 1,
+    }
+  }
+}
+
+const chainMock = {
+  getEventHash(): string {
+    return 'event hash';
+  },
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
same thing here super nitpick, could just do general disable on this lint rule for these mocks